### PR TITLE
Feature/update bolt

### PR DESCRIPTION
**Pull request by brandonroot**
# Overview
Git reset dev to be even with master (pre native changes)
The update bolt introduces the ability to update or update with script (which are individual bolts)

## Testing

lein test 
Run locally, all tests pass

## Reviewers
@geodrome

### CODE REVIEW

File: src/elasticsearch_bolt/core.clj [hunk 1]
```diff
@@ -29,6 +31,13 @@
 (defn get-doc [index doctype id]
   (esd/get index doctype id) )
 
+(defn get-doc [index doctype id]
```
Comment (geodrome) [inline src/elasticsearch_bolt/core.clj]: is get-doc defined twice?

File: src/elasticsearch_bolt/core.clj [hunk 2]
```diff
@@ -47,9 +56,26 @@
     (bulk/bulk (bulk/bulk-index prepped-docs)
                :refresh true)))
 
-(defn index-doc [index doctype id doc]
-  (esd/put index doctype id doc))
```
Comment (geodrome) [inline src/elasticsearch_bolt/core.clj]: `index-doc` not needed?

Comment (brandonroot) [inline src/elasticsearch_bolt/core.clj]: it isn't referenced anywhere, but there isn't anything wrong with it, i'll add it back in

File: src/elasticsearch_bolt/core.clj (hunk 1 above)
Comment (brandonroot) [inline src/elasticsearch_bolt/core.clj]: Oops

**Pull request merged**